depinject/debug.go

```diff
@@ -79,14 +100,17 @@ func Logger(logger func(string)) DebugOption {
 	})
 }
 
-const debugContainerDot = "debug_container.dot"
+const (
+	debugContainerDot = "debug_container.dot"
+	debugContainerLog = "debug_container.log"
```

Comment:
nit: can this file be added into the gitignore.

Reply:
Yes. But I'm also not seeing these files getting generated at all... even the .dot file which was supposed to be there before.

Reply:
Moved the depinject stuff to #14825